packages/hash/frontend/src/blocks/page/PageTitle.tsx

```diff
@@ -70,16 +94,34 @@ export const PageTitle: FunctionComponent<PageTitleProps> = ({
 
   // TODO: Assign appropriate a11y attributes
   return (
-    <input
-      placeholder="A title for the page"
-      disabled={updateEntityLoading}
-      onChange={handleInputChange}
+    <Typography
+      id="hash-page-title"
+      component="h1"
+      variant="h2"
+      contentEditable={!updateEntityLoading}
+      suppressContentEditableWarning
       onKeyDown={handleInputKeyDown}
       onBlur={handleInputBlur}
-      className={tw`border-none font-medium text-2xl w-full py-0.5 -mx-1 px-1 mt-px ${
-        updateEntityLoading ? "opacity-50" : undefined
-      }`}
-      value={inputValue}
-    />
+      onInput={handleInputChange}
+      sx={{
+        "*": {
```

Comment:
It exists on `variant="h2"`

Reply:
When running the app I'm seeing `line-height` of `46.872px` for the title.

![image](https://user-images.githubusercontent.com/37453800/186627469-a95f10cd-8000-4348-b6b4-4af0f9f947ef.png)

Reply:
Ooh, I realized design has an overrided line height on top of h2 style, you're right. Updated it here https://github.com/hashintel/hash/pull/956/commits/c6ee895dbf23e35eaf301b83fdd5088e37c87355

Reply:
Just to add, it's normal for the line-height to have that value. This is because we make use of fluid typography for font sizes (font size changes responsively based on the screen size). Here's the typescale we use for this => https://utopia.fyi/type/calculator/?c=320,15,1.2,1200,16,1.25,6,2,&s=0.75%7C0.5%7C0.25,1.5%7C2%7C3%7C4%7C6,s-l. 

Here's our implementation in `hash-design-system` => https://github.com/hashintel/hash/blob/main/packages/hash/design-system/src/theme/components/utils/mui-css-baseline-theme-options.ts. 

If the font-size is 37.32px and line-height is 1.2, the computed line-height will be 44.784.

Reply:
Thanks for reminding @teenoh! I was aware of that, but the title user in `PageTitle` has even a different line-height compared to the `h2` base on design-system. On design-system it was `1.2`, but on this component, it's `1.23` on design.